### ISSUE TITLE
deployment works bad with config - projectLocation

### ISSUE THREAD

**Issue by mocenas** [bucharest-gold/nodeshift#188]
Some phases of the build ignores property _projectLocation_ and runs against current working dir.
example:
```
nodeshift.deploy{
            'projectLocation': 'different/dir/than/current/',
            'strictSSL': false
        })
```
output:
```
...
WARNING The following files do not exist: bin,lib,public,app.js,licenses
...
```
although all of those files exists in the specified dir.

When I bypassed the code that checks for file existence, another error occured.
Workaround is to change original code to:
 ```
process.chdir('different/dir/than/current/');
nodeshift.deploy{
            'projectLocation': 'different/dir/than/current/',
            'strictSSL': false
        })
```

**Comment by lholmquist** [comment]
@mocenas what is the sample project you are using to test this with?

**Comment by mocenas** [comment]
I'am using it at https://github.com/bucharest-gold/nodejs-circuit-breaker to test circuit break, running both services simultaneously. 
I didn't commited the problematic part of the code to any public repo yet.

**Comment by lholmquist** [comment]
if you could commit the problem part, so i can see whats going on, that would be helpful.  I think i have an idea of where the issue is

**Comment by mocenas** [comment]
Ok commited to my repo:
https://github.com/mocenas/nodejs-circuit-breaker
in file:
https://github.com/mocenas/nodejs-circuit-breaker/blob/master/test/openshift-test.js

**Comment by lholmquist** [comment]
thanks,  and a tip.  you should use branches instead of committing to your master branch

**Comment by lholmquist** [comment]
i think i found where nodeshift needs to be update.

**Comment by lance** [comment]
@mocenas just FYI, I already have an integration test PR for circuit breaker here: https://github.com/bucharest-gold/nodejs-circuit-breaker/pull/27.

**Comment by lholmquist** [comment]
@mocenas just released version 1.3.2 that should fix the issue
